arch/arm/core/cortex_m/mpu/arm_mpu_v8_internal.h

```diff
@@ -9,6 +9,9 @@
 #define ZEPHYR_ARCH_ARM_CORE_CORTEX_M_MPU_ARM_MPU_V8_INTERNAL_H_
 
 #include <cmse.h>
+#define LOG_LEVEL CONFIG_ARCH_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_DECLARE(mpu);
```

Comment:
Unfortunately, you cannot declare/register module in header file. It will end in conflict when you try to include such file in C file which declares logger module too.